boards/arm/mimx8mm_phyboard_polis/mimx8mm_phyboard_polis_defconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 PHYTEC Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8MM_M4=y
+CONFIG_SOC_MIMX8MM6=y
+CONFIG_BOARD_MIMX8MM_PHYBOARD_POLIS=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+
+# config needed to not generate an error, when using remoteproc
+CONFIG_OPENAMP=y
+CONFIG_OPENAMP_RSC_TABLE=y
+
+# has to be y, if M4's TMC should be used
+CONFIG_XIP=y
```

Comment:
You have defaults in the board `Kconfig.defconfig` for when XIP is disabled, but are requiring it be enabled here. Does this board support running with `CONFIG_XIP=n`?